Makefile

```diff
@@ -171,6 +171,7 @@ test: run_test coverage vg coverage-check vg-check
 # Ignore failure here; it will be detected by Jenkins.
 .PHONY: run_test
 run_test: build_test | $(TEST_OUT_DIR)
+	mkdir -p /var/run/clearwater
```

Comment:
How do the stats unit tests run? They should have the same problem, so I should probably do what they do, but I'm not sure where they are located.